packages/starlight-cooler-credit/translations.ts

```diff
@@ -33,12 +33,19 @@ export const Translations = {
         "starlightCoolerCredit.astro.description": "Vuoi creare il tuo sito web statico? →",
         "starlightCoolerCredit.starlight-blog.title": "Creato con Starlight Blog",
         "starlightCoolerCredit.starlight-blog.description": "Vuoi creare il tuo blog Starlight? →",
-    },    
+    },
     ru: {
         "starlightCoolerCredit.starlight.description": "Хотите создать свою документацию? →",
         "starlightCoolerCredit.astro.title": "Создано с помощью Astro",
         "starlightCoolerCredit.astro.description": "Хотите создать свой статичный сайт? →",
         "starlightCoolerCredit.starlight-blog.title": "Создано с помощью Starlight Blog",
         "starlightCoolerCredit.starlight-blog.description": "Хотите создать свой Starlight блог? →",
     },
+    ja: {
+        "starlightCoolerCredit.starlight.description": "自分のドキュメントを作成したいですか?→",
```

Comment:
It might be better to add a space before the arrow to match the others.

```suggestion
        "starlightCoolerCredit.starlight.description": "自分のドキュメントを作成したいですか? →",
```